src/systemcmds/netman/netman.cpp

```diff
@@ -405,12 +405,22 @@ static void usage(const char *reason)
   memory. On boot the `update` option will be run. If a network configuration
   does not exist. The default setting will be saved in non-volatile and the
   system rebooted.
-  On Subsequent boots, the `update` option will check for the existence of
-  `net.cfg` in the root of the SD Card.  It will saves the network settings
-  from `net.cfg` in non-volatile memory, delete the file and reboot the system.
-
-  The `save` option will `net.cfg` on the SD Card. Use this to edit the settings.
-  The  `show` option will display the network settings  to the console.
+  
+  ### update
+  `netman update` is run automatically by the startup script. When run,
+  the `update` option will check for the existence of `net.cfg` in the
+  root of the SD Card. It will saves the network settings from `net.cfg`
+  in non-volatile memory, delete the file and reboot the system.
+
+  ### save
+  The `save` option will save settings from non-volatile memory to a file named
+  `net.cfg` on the SD Card filesystem for editing. Use this to edit the settings. 
+  Save does not immediately apply the network settings; it depends on the user
+  rebooting; Where the `update` command is run by the start-up script and also saves the settings.
+  
+    
+  ### show
+  The `show` option will display the network settings in `net.cfg` to the console.
```

Comment:
1. The headings should be part of description.
2. Is this a particular startup script, such as extras.txt, a board config, an airframe config, or just "some part of the startup"?
   If the later, we might change to from "the startup script" to as shown below

```suggestion
  #### update
  
  `netman update` is run automatically by [a startup script](../concept/system_startup.md#system-startup).
  When run, the `update` option will check for the existence of `net.cfg` in the root of the SD Card.
  It then saves the network settings from `net.cfg` in non-volatile memory, 
  deletes the file and reboots the system.

  #### save
  
  The `save` option will save settings from non-volatile memory to a file named
  `net.cfg` on the SD Card filesystem for editing. Use this to edit the settings. 
  Save does not immediately apply the network settings; the user must reboot the flight stack.
  By contrast, the `update` command is run by the start-up script and also saves the settings.
  
  #### show
  
  The `show` option will display the network settings in `net.cfg` to the console.
```

Reply:
change `also saves the settings.` to `and commits the settings to non-volatile memory, reboots the flight controller which will then use the new settings.`

Reply:
Thanks @davids5 done, and I'll commit.